ff/serving/provider/offline_test.go

```diff
@@ -44,18 +58,24 @@ func TestOfflineStores(t *testing.T) {
 	}
 }
 
-func randomResourceID() ResourceID {
-	possibleTypes := []OfflineResourceType{Label, Feature}
-	randType := possibleTypes[rand.Intn(2)]
+func randomID(types ...OfflineResourceType) ResourceID {
+	var t OfflineResourceType
+	if len(types) == 0 {
+		t = NoType
+	} else if len(types) == 1 {
```

Comment:
else if and else can be combined since they perform the same logic, t = types[rand.Intn(len(types))]

Reply:
true, but I'd argue it a little clearer this way